tests/test_physical_systems/test_voltage_supplies.py

```diff
@@ -95,7 +96,131 @@ def test_system_equation(self):
          #time invariance
          assert system_equation(0,[30],50,3,3,3) == system_equation(5,[30],50,3,3,3)     
 
+class TestAC1PhaseSupply(TestVoltageSupply):
+    key = 'AC1PhaseSupply'
+    class_to_test = vs.AC1PhaseSupply
+    
+    def test_default_initialization(self):
+        """Test for default initialization values"""
+        voltage_supply = vs.AC1PhaseSupply()
+        assert voltage_supply.u_nominal == 230.0
+        assert voltage_supply._max_amp == 230.0 * np.sqrt(2)
+        assert voltage_supply.supply_range == [-230.0 * np.sqrt(2), 230.0 * np.sqrt(2)]
+        assert voltage_supply._fixed_phi == False
+        assert voltage_supply._f == 50
+        
+    def test_reset(self, u_nominal=230.0):
+        """Test the reset function for correct behavior on fixed phase""" 
+        supply_parameter = {'frequency': 50, 'phase': 0, 'fixed_phase': False}
+        voltage_supply = vs.AC1PhaseSupply(u_nominal, supply_parameter)
+        assert voltage_supply._phi == 0
+        _ = voltage_supply.reset()
+        assert voltage_supply._phi != 0, "Test this again and if this error doesn't appear next time you should consider playing lotto"
 
+        supply_parameter = {'frequency': 50, 'phase': 0, 'fixed_phase': True}
+        voltage_supply = vs.AC1PhaseSupply(u_nominal, supply_parameter)
+        assert voltage_supply._phi == 0
+        assert voltage_supply.reset() == [0.0]
+        assert voltage_supply._phi == 0
 
+        
+    def test_initialization(self, u_nominal = 300.0):
+        supply_parameter = {'frequency': 35, 'phase': 0, 'fixed_phase': True}
+        voltage_supply = vs.AC1PhaseSupply(u_nominal, supply_parameter)
+        assert voltage_supply.u_nominal == 300.0
+        assert voltage_supply._max_amp == 300.0 * np.sqrt(2)
+        assert voltage_supply.supply_range == [-300.0 * np.sqrt(2), 300.0 * np.sqrt(2)]
+        assert voltage_supply._fixed_phi == True
+        assert voltage_supply._phi == 0
+        assert voltage_supply._f == 35
+        
+    def test_get_voltage(self):
+        """Test the get voltage function for different times t."""
+        supply_parameter = {'frequency': 1, 'phase': 0, 'fixed_phase': True}
+        supply = vs.AC1PhaseSupply(supply_parameter=supply_parameter)
+        
+        #Test for default sinus values
+        times = [0, 2*np.pi, 4*np.pi]
+        for time in times:
+            assert supply.get_voltage(time) == pytest.approx([0.0])
```

Comment:
Please have a look at 
https://numpy.org/doc/stable/reference/generated/numpy.allclose.html
Importing pytest for approximate comparisons is not necessary.